k2/csrc/ragged_ops.cu

```diff
@@ -1379,4 +1380,14 @@ Array1<int32_t> GetDecreasingSizeOrder(RaggedShape &shape) {
   return index_map;
 }
 
+RaggedShape GetLayer(RaggedShape &src, int32_t layer) {
+  K2_CHECK_GE(layer, 0);
```

Comment:
also check `src.NumAxes() >=2`?

Reply:
Mm, I don't always check this because such a Ragged objecct would not be valid, and for instance probably could not be constructed from Python.

Reply:
OK